rllib/evaluation/worker_set.py

```diff
@@ -222,21 +229,16 @@ def add_workers(self, num_workers: int) -> None:
             num_workers: The number of remote Workers to add to this
                 WorkerSet.
         """
-        remote_args = {
-            "num_cpus": self._remote_config["num_cpus_per_worker"],
-            "num_gpus": self._remote_config["num_gpus_per_worker"],
-            "resources": self._remote_config["custom_resources_per_worker"],
-        }
-        cls = RolloutWorker.as_remote(**remote_args).remote
+        old_num_workers = len(self._remote_workers)
         self._remote_workers.extend(
             [
                 self._make_worker(
-                    cls=cls,
+                    cls=self._cls,
                     env_creator=self._env_creator,
                     validate_env=None,
                     policy_cls=self._policy_class,
-                    worker_index=i + 1,
-                    num_workers=num_workers,
+                    worker_index=old_num_workers + i + 1,
```

Comment:
I don't think you want to +1 here?
if there are 5 old_num_workers, you want the next worker_index to be 5.

Reply:
Nope, 6 is ok. Index 0 is reserved for the local worker, so remote workers start from index=1. `i` here runs from 0 up.

Reply:
get it.